ee/clickhouse/queries/trends/trend_event_query.py

```diff
@@ -54,8 +55,18 @@ def get_query(self) -> Tuple[str, Dict[str, Any]]:
         date_query, date_params = self._get_date_filter()
         self.params.update(date_params)
 
-        prop_filters = [*self._filter.properties, *self._entity.properties]
-        prop_query, prop_params = self._get_props(prop_filters)
+        if self._filter.properties:
```

Comment:
makes sense that property filters will always be AND'ed with entity filters, and we don't have to worry about max-2-levels, as test-account-filters can come in as well.

Was thinking, since this logic is the same in every event_query, we ought to abstract it out?

I somewhat fear that this stuff (event query classes) is getting wayy too complex to keep in one head. (just a side-observation that makes me uncomfortable, not sure there's anything to do about it)

Reply:
It's only needed in trends and stickiness event query (for now)